.test-defs/infrastructure-test.yaml

```diff
@@ -0,0 +1,23 @@
+kind: TestDefinition
+metadata:
+  name: infrastructure-test
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Integration test for infrastructure creation and deletion
+  activeDeadlineSeconds: 1800
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/integration/infrastructure
+    --v -ginkgo.v -ginkgo.progress -ginkgo.noColor
+    --kubeconfig=$TM_KUBECONFIG_PATH/testmachinery.config
+    --auth-url='$(shell cat $(AUTH_URL))' \
```

Comment:
You are missing one more file that ultimately will also decide how the tests are  configured and whether you can read these  params from a file or env. 

You can see an example here: https://github.com/gardener/gardener-extension-provider-aws/blob/master/.ci/testruns/default.yaml